test/models/questionnaire_test.rb

```diff
@@ -103,4 +103,79 @@ class QuestionnaireTest < ActiveSupport::TestCase
 
     assert_error @questionnaire, :base, :cannot_destroy_with_answered_poll
   end
+
+  test 'allows clearing pollable_type when never used' do
+    q = Questionnaire.list.create!(
+      name: 'Unused questionnaire',
+      organization_id: organizations(:cirope).id,
+      email_subject: 'subject',
+      email_text: 'text',
+      email_link: 'link',
+      questions_attributes: {
+        '1' => {
+          question: 'Question',
+          sort_order: 1,
+          answer_type: 0
+        }
+      }
+    )
+
+    assert q.update(pollable_type: 'ConclusionReview'),
+           q.errors.full_messages.join('; ')
+    assert_equal 'ConclusionReview', q.reload.pollable_type
+
+    assert q.update(pollable_type: nil),
+           q.errors.full_messages.join('; ')
+    assert_nil q.reload.pollable_type
+  end
+
+  test 'blocks clearing pollable_type when already used in conclusion final review' do
+    @questionnaire.update! pollable_type: 'ConclusionReview'
+    @questionnaire.polls.update_all pollable_type: 'ConclusionReview'
+
+    assert_not @questionnaire.update pollable_type: nil
```

Comment:
```suggestion
    refute @questionnaire.update pollable_type: nil
```
Solo porque se lee más alegre.